common/src/main/resources/locale/messages-en_US.properties

```diff
@@ -153,3 +153,6 @@ integrations.towny.cannot_use_alliance_channel=<red>You must join an alliance to
 integrations.towny.cannot_use_nation_channel=<red>You must join a nation to use this channel.
 integrations.towny.cannot_use_town_channel=<red>You must join a town to use this channel.
 integrations.mcmmo.cannot_use_party_channel=<red>You must join an mcMMO party to use this channel.
+integrations.fuuid.cannot_use_faction_channel=<red>You must join a faction to use this channel.
+integrations.fuuid.cannot_use_alliance_channel=<red>You must join an alliance to use this channel.
```

Comment:
all 3 channels are checking the same thing, whether the player is in a faction, is that intentional? if so probably change these messages to reflect that. (should keep them separate even if the message is the same to allow changing them separately)

Reply:
The alliance/truce channels now check if there are any factions allied/truced with the players'